enterprise/api_client/enterprise.py

```diff
@@ -55,7 +55,7 @@ def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
                 content_id = utils.get_content_metadata_item_id(item)
                 content_metadata[content_id] = item
 
-        return content_metadata.values()
+        return list(content_metadata.values())
```

Comment:
?

Reply:
dict.values() in python 2.7 returns a list containing values but in python 3 it returns object of dict_values which behave like set.

https://www.python.org/dev/peps/pep-3106/

Reply:
@iamsobanjaved We already had tests running on python3.6 and passing. Which is similar to py3.5 dict.values(). So why we need to type cast now.